drivers/at_cmd/at_cmd.c

```diff
@@ -307,7 +307,7 @@ void at_cmd_set_notification_handler(at_cmd_handler_t handler)
 	k_mutex_unlock(&cmd_pending);
 }
 
-static int at_cmd_init(struct device *dev)
+static int at_cmd_driver_init(struct device *dev)
```

Comment:
The init function was not public before. We should now maybe consider returning errno and not just `-1` in case of error? Could be useful for the user to get the error reason.
This was one of my comment when the module was created. Since it was not a public function, we did not consider it.

Reply:
👍 done